### ISSUE TITLE
add support for dual gzip/zstd image support

### ISSUE THREAD

**Issue by bsherman** [ublue-os/image-template#24]
placeholder to match/support efforts ongoing in main repo.

**Comment by castrojo** [comment]
tldr the version of rpm-ostree in F39 is too old and will error out when it sees a zstd compressed image. 

p5 is in progress of adding support to our workflows to push both gzip and zstd to the same image. Fedora will publish zstd:chunked by default for F41, but they are just now switching to OCI so they don't have previous users to transition. Bluefin will push both gzip/zstd in F40 to slurp up upgrading users from GTS, and then F41 will be zstd:chunked only. 

We should encourage zstd:chunked as the default, but should probably have the option to push to both with comments so people don't end up in a situation where their older custom image can't upgrade.

**Comment by Venefilyn** [comment]
Feels like I'm [encountering this issue](https://github.com/Venefilyn/VeneOS/actions/runs/11571680640/job/32226618742), how would one enable this exactly? Is it just about enabling oci for the `redhat-actions/buildah-build@v2` action?

**Comment by xlionjuan** [comment]
@Venefilyn Change runner to `ubuntu-24.04`, `ubuntu-latest` is still using 22.04, and 22.04's Podman didn't support zstd.

**Comment by castrojo** [comment]
I think by the time zstd chunked lands we're not going to care about F39. I don't think we need this.
